hello_world/blockchain_app/index.js

```diff
@@ -0,0 +1,41 @@
+const { Application, configDevnet, utils, HTTPAPIPlugin} = require('lisk-sdk');
+const genesisBlockDevnet = require('./genesis-block');
```

Comment:
Did you checked with latest `lisk-sdk` genesis block, is it really not working?

Reply:
Yes, I checked with latest SDK